website/docs/using-odo/json-output.md

```diff
@@ -0,0 +1,43 @@
+---
+title: JSON Output
+sidebar_position: 2
+---
+# JSON Output
+
+The `odo` commands that output some content generally accept a `-o json` flag to output this content in a JSON format, suitable for other programs to parse this output more easily.
+
+The output structure is similar to Kubernetes resources, with `Kind`, `apiVersion`, `metadata` ,`spec` and `status` fields.
+
+List commands return a `List` resource, containing an `items` (or similar) field listing the items of the list, each item being also similar to Kubernetes resources.
+
+Delete commands return a `Status` resource; see the [Status Kubernetes resource](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/status/).
+
+Other commands return a resource associated with the command (`Application`, `Storage`', `URL`, etc).
+
+The exhaustive list of commands accepting the `-o json` flag is currently:
+
+| commands                       | Kind (version)                          | Kind (version) of list items                                 | Complete content?         | 
```

Comment:
What does "Complete content" mean? Are there any commands where it's incomplete? I'd expect this to be binary - either JSON output is present or not present. But "Complete" makes me think it's about something else.

Reply:
Does the content contains all the data present in the human readale output (or more), and is equivalent to it.